src/modules/Lan.rb

```diff
@@ -660,7 +661,8 @@ def Packages
         pkgs << "NetworkManager" if !PackageSystem.Installed("NetworkManager")
       elsif !PackageSystem.Installed("wpa_supplicant")
         # we have to add wpa_supplicant when wlan is in game, wicked relies on it
-        pkgs << "wpa_supplicant" if !LanItems.find_type_ifaces("wlan").empty?
+        wlan_present = yast_config.interfaces.any? { |i| i.type == Y2Network::InterfaceType::WIRELESS }
```

Comment:
This will look for wireless cards but there is no check if them are configured or not. So, it should check for connections instead of interfaces.

Reply:
sure. It works according to assumption "if there is wireless card in the system, wpa_supplicant might be useful". And it obviously preserves old behavior. Do we want to change it.

Reply:
I remembered it was only installed if there was some "wlan" interface configured but not otherwise. That was the behavior until https://github.com/yast/yast-network/pull/683

Reply:
https://github.com/yast/yast-network/commit/5e3ad1a6b2b9a3261a18c680e34a06a142bcb475#diff-c536d97bfa2e800b076011ec2976dae8b9c77b83fec29c7a1741554d1f8750c0R991

Reply:
https://github.com/yast/yast-network/pull/688